src/Program.cs

```diff
@@ -67,6 +67,13 @@ public static async Task<int> Run(string workspace, string verbosity, bool dryRu
                 // Since we are running as a dotnet tool we should be able to find an instance of
                 // MSBuild in a .NET Core SDK.
                 var msBuildInstance = Build.Locator.MSBuildLocator.QueryVisualStudioInstances().First();
+
+                // Since we do not inherit binding redirects from dotnet when referencing the SDK copy

```

Comment:
"binding redirects" is a .NET Framework concept. I would say "since we do not inherit msbuild.deps.json."